server/views/pages/recommendations/searchPpud.njk

```diff
@@ -23,31 +23,39 @@
                         Name
                     </dt>
                     <dd class="govuk-summary-list__value">
+                    {% if recommendation.personOnProbation.fullName %}
                           {{ recommendation.personOnProbation.fullName }}
+                    {% else %}-{% endif %}
```

Comment:
Should these hyphens include the screen reader text saying "no value" or something similar?

Reply:
yes, hyphens should include the screen reader text saying "no value"

Reply:
Done